src/components/breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './Breadcrumbs.css'
+
+function makeBread (root) {
+  if (root.endsWith('/'))
+    root = root.substring(0, root.length - 1)
+
+  if (root.startsWith('/'))
+    root = root.substring(1, root.length)
+
+  let parts = root.split('/').map(part => {
+    return {
+      name: part,
+      path: part
+    }
+  })
+
+  parts[0].path = `/${parts[0].path}`
+
+  for (let i = 1; i < parts.length; i++) {
+    parts[i] = {
+      name: parts[i].name,
+      path: parts[i - 1].path + '/' + parts[i].path
+    }
+  }
+
+  return parts
+}
+
+export default function Breadcrumbs ({path, onClick, className = '', ...props}) {
```

Comment:
We need to think about about the API for this component. we'll have addresses like
`/ipfs/QmHash/foo/bar`. The simplest api would let us pass a valid ipfs address to the breadcrumb and have it do the right thing. I don't see the breadcrumb as a generic component like button or checkbox is, so let's move it to the files directory, and make it a bit less generic.

We'll probably represent the users MFS root with house icon. We won't show the label "MFS root" as that is an internal implementation detail.

Reply:
![image](https://user-images.githubusercontent.com/5447088/40229335-d8888f44-5a8b-11e8-8827-016921c8a890.png)

@olizilla yeah, I agree. So here we would remove the IPFS and MFS Root and put a home thumbnail right?